pretix_mollie/payment.py

```diff
@@ -1389,12 +1382,6 @@ class MolliePaysafecard(MolliePaymentMethod):
     refunds_allowed = False
 
 
-class MollieSofort(MolliePaymentMethod):
-    method = "sofort"
-    verbose_name = _("SOFORT via Mollie")
-    public_name = _("SOFORT (instant bank transfer)")
-
```

Comment:
Instead of deleting the class, only add these, so old records don't break:


```
def is_allowed(self, request: HttpRequest, total: Decimal = None) -> bool:
        # Mollie<>Sofort is shut down since September 30th 2024
        return False

def order_change_allowed(self, order: Order, request: HttpRequest = None) -> bool:
        # Mollie<>Sofort is shut down since September 30th 2024
        return False
```